src/control/server/drpc.go

```diff
@@ -46,17 +46,26 @@ func getDrpcClientConnection(sockDir string) *drpc.ClientConnection {
 // drpcSetup creates socket directory, specifies socket path and then
 // starts drpc server.
 func drpcSetup(sockDir string, iosrv *iosrv) error {
+	var msg string
+
```

Comment:
msg could be restricted to the scope of "{" on new line 53.

Reply:
done